### PR TITLE
Added devnet32 and mainnet15 to sporks json

### PR DESCRIPTION
**Pull request by franklywatson**
Updated git commit hash after tag updated to v0.23.4 for mainnet15

Closes: #???

## Description

Updated to capture recent spork changes

______

For contributor use:

- [X] Targeted PR against `master` branch
- [X] Updated relevant documentation

### CODE REVIEW

File: sporks.json [hunk 1]
```diff
@@ -3,6 +3,34 @@
...
+          "root-state-commit": "84d9b325d3f48f7f075eea0db6c246cb5adc759a2dd8561e461477a7ca1f7f22",
+          "git-commit-hash": "bfde54ae3324db5d18ebeaa22c70b5574a114f2e",
+          "state-artefacts" : {
+            "gcp": {
+              "root-checkpoint-file": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-15-execution/public-root-information/root.checkpoint",
+              "root-protocol-state-snapshot": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-15-execution/public-root-information/root-protocol-state-snapshot.json",
+              "node-info": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-15-execution/public-root-information/node-infos.pub.json",
+              "execution-state-wal-chunks": "https://storage.googleapis.com/flow_public_mainnet15_execution_state"
+            },
+            "s3": {
```
Comment (vishalchangrani) [inline sporks.json:21]: there is an AWS s3 bucket now `flow-public-mainnet15-execution-state` (same name as GCP but dashes instead of underscores)

Comment (vishalchangrani) [reply sporks.json:21]: for the execution state wal chunks (not for anything else)

Comment (franklywatson) [reply sporks.json:21]: Oh great! Let me check on the contents, thanks

Comment (franklywatson) [reply sporks.json:21]: @vishalchangrani added the EXE state bucket for mainnet15, but the other buckets aren't yet spun up and populated

**Review by vishalchangrani — APPROVED** [review]

**Pull request merged**